src/main/java/tic/tac/toe/Board.java

```diff
@@ -14,6 +14,9 @@ public int getBoardSize() {
   }
 
   public boolean isCellEmpty(int cell) {
+    if (cell < 1 || cell > board.length) {
+      return false; // non-existant cells are not empty
```

Comment:
remove comment, perhaps refactor to instead of false be 'empty cell' which maps to false

Reply:
done